source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/APIControllers/Groups/GetGroupTest.cs

```diff
@@ -0,0 +1,152 @@
+
+        private void BuildTestData()
+        {
+            CurrentUser = NeedsCurrentUser();
+            org1 = AddEntity<AppDbContext, Organization>(new Organization
+            {
+                Name = "TestOrg1",
+                WebsiteUrl = "https://testorg1.org",
+                BuildEngineUrl = "https://buildengine.testorg1",
+                BuildEngineApiAccessToken = "replace",
+                OwnerId = CurrentUser.Id
+
+            });
+            org2 = AddEntity<AppDbContext, Organization>(new Organization
+            {
+                Name = "TestOrg2",
+                WebsiteUrl = "https://testorg2.org",
+                BuildEngineUrl = "https://buildengine.testorg2",
+                BuildEngineApiAccessToken = "replace",
+                OwnerId = CurrentUser.Id
+
+            });
+            org3 = AddEntity<AppDbContext, Organization>(new Organization
+            {
+                Name = "TestOrg3",
+                WebsiteUrl = "https://testorg3.org",
+                BuildEngineUrl = "https://buildengine.testorg3",
+                BuildEngineApiAccessToken = "replace",
+                OwnerId = CurrentUser.Id
+
+            });
+            CurrentUserMembership = AddEntity<AppDbContext, OrganizationMembership>(new OrganizationMembership
+            {
+                UserId = CurrentUser.Id,
+                OrganizationId = org1.Id
+            });
+            CurrentUserMembership2 = AddEntity<AppDbContext, OrganizationMembership>(new OrganizationMembership
+            {
+                UserId = CurrentUser.Id,
+                OrganizationId = org2.Id
+            });
+            group1 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup1",
+                Abbreviation = "TG1",
+                OwnerId = org1.Id
+            });
+            group2 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup2",
+                Abbreviation = "TG2",
+                OwnerId = org1.Id
+            });
+            group3 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup3",
+                Abbreviation = "TG3",
+                OwnerId = org2.Id
+            });
+            group4 = AddEntity<AppDbContext, Group>(new Group
+            {
+                Name = "TestGroup4",
+                Abbreviation = "TG4",
+                OwnerId = org3.Id
+            });
+            groupMembership1 = AddEntity<AppDbContext, GroupMembership>(new GroupMembership
+            {
+                UserId = CurrentUser.Id,
+                GroupId = group1.Id
+            });
+        }
+
+        [Fact]
+        public async Task Get_Group_With_An_OrganizationHeader()
+        {
+            BuildTestData();
+
+            var url = "/api/groups/" + group1.Id.ToString();
+            var response = await Get(url, org1.Id.ToString());
```

Comment:
I wonder if `Get` should be able to take an Integer and automatically call `.ToString()` for you